docs/eventing/getting-started.md

```diff
@@ -1,128 +1,66 @@
-eventing resources you need to manage events.
+You use namespaces to group together and organize your Knative
+resources.
+<!--TODO: Add documentation about namespaces to core docs?-->
 
-#### Optional Sugar Controller
+1. Create a namespace called `event-example` by entering the following command:
 
-When installed, the
-[Sugar Controller for Knative Eventing](../install#optional-eventing-extensions)
-will add a controller that reacts to special labels and annotations for your
-resources, you can skip manually creating a Broker in the above step and instead
-label a namespace:
+  ```
+  kubectl create namespace event-example
+  ```
+This creates an empty namespace called `event-example`.
 
-```sh
-kubectl label namespace event-example eventing.knative.dev/injection=enabled
-```
-
-This gives the `event-example` namespace the `eventing.knative.dev/injection`
-label, which adds resources that will allow you to manage your events.
-
-### Validating that the `Broker` is running
+### Add a broker to the namespace
 
-The [`Broker`](./broker/README.md#broker) ensures that every event sent by event
-producers arrives at the correct event consumers. The `Broker` was created when
-you labeled your namespace as ready for eventing, but it is important to verify
-that your `Broker` is working correctly. In this guide, you will use the default
-broker.
+The [`broker`](./broker/README.md#broker) allows you to route events to different event sinks or consumers.
 
-1. Run the following command to verify that the `Broker` is in a healthy state:
+1. Add a broker named `default` to your namespace by entering the following command:
 
-   ```sh
-   kubectl --namespace event-example get Broker default
-   ```
-
-   This shows the `Broker` that you created:
+  ```
+  kubectl create -f - <<EOF
+  apiVersion: eventing.knative.dev/v1
+  kind: Broker
+  metadata:
+   name: default
+   namespace: event-example
+  EOF
+  ```
 
-   ```sh
-   NAME      READY   REASON   URL                                                        AGE
-   default   True             http://default-broker.event-example.svc.cluster.local      1m
-   ```
+1. Verify that the broker is working correctly, by entering the following command:
 
-   When the `Broker` has the `READY=True` state, it can begin to manage any
-   events it receives.
+  ```
+  kubectl --namespace event-example get Broker default
+  ```
+This shows information about your broker. If the broker is working correctly, it shows a `READY` status of `True`:
 
-1. If `READY=False`, wait 2 minutes and re-run the command. If you continue to
-   receive the `READY=False`, see the [Debugging Guide](./debugging/README.md)
-   to help troubleshoot the issue.
+  ```
+  NAME      READY   REASON   URL                                                        AGE
+  default   True             http://default-broker.event-example.svc.cluster.local      1m
+  ```
 
-Now that your `Broker` is ready to manage events, you can create and configure
```

Comment:
I really think this 2 minutes comes from previous incarnations where there were many resources being created, etc. If the Broker doesn't come ready rerunning the command doesn't help anything. I would maybe just say that if Broker doesn't come ready (in the order that all of our resources come, serving included), we should jump to debugging guide. I don't know how we word those in all our resources, point is just that nobody should wait for 2 minutes, and re-running won't fix anything :) I'll stop yammering :)

Reply:
Makes sense to me, I've removed the mention of 2 mins now.

Reply:
I think you removed it from different place? Still shows it here? :)